BrentOzarULTD/SQL-Server-First-Responder-Kit
# sp_DatabaseRestore - Fix @MoveFile dependency bug Issue 1192 #1192

Fixes #1192

Changes proposed in this pull request:
Derive values for @MoveDataDrive/@movelogdrive from SERVERPROPERTY if not specified

How to test this code:
- Try the 4 combinations of @MoveDataDrive/@movelogdrive
- Specify odd non-default paths and check sys.master_files at each turn

Has been tested on (remove any that don't apply):
 - SQL Server 2014
 - SQL Server 2016
